readme.md

```diff
@@ -1,32 +1,29 @@
-# Intermediate Git
+# Git-Branching
 
 ## Learning Objectives
 
 - Explain what a branch is in git
 - Create, merge and delete branches on local and remote repositories
 - Describe how branching and merging allows for collaboration during development
+- Describe Github Workflows using issues, branches, and pull-requests, and merge conflicts
 - Resolve a merge conflict
 
-
-## Review Git Basics (5 min)
+## Opening Framing (5 min)
 
 Quickly review the basics of git:
 
-1. What is the most common workflow for creating save points while working
+1. What is the most common workflow for committing while working
 locally?
-2. What commands are used to share changes (commits) between repos?
-3. Describe the fork/clone model, and how it is used for HW submission.
-
-## You Do: Branching Overview (15 min)
+2. What commands are used to share changes (commits) between local and remote repos?
+3. Describe the differences between a fork and a clone.
+4. What are the differences between Git and Github?
 
-We are going to start with a [brief tutorial](http://pcottle.github.io/learnGitBranching/).  This is an introduction to branching.
```

Comment:
before the you-do, I recommend framing what branching is and why we use it. For example:

>a git branch is a label for a commit. We use branches to make experimental changes. 

Also, feel free to include my work experience as your own:

One time, I was working on a new feature for a client, committing directly to master. The client found a typo issue, and I had to fix it immediately. Because i was committing to master, I had to undo my work in order to fix and ship the typo...